docs/src/main/paradox/lagom/overview.md

```diff
@@ -0,0 +1,48 @@
+# Lagom gRPC support overview
+
+## gRPC
+
+[gRPC](https://grpc.io/) is a transport mechanism for request/response and (non-persistent) streaming use cases. See [Akka gRPC documentation for an explanation of why gRPC](https://developer.lightbend.com/docs/akka-grpc/current/whygrpc.html) and when to use it as your transport.
+
+## How to add the sbt plugin
+
+To get started, for both client and server code generation, you will need to add Akka gRPC sbt plugin:
+
+sbt
+:   @@@vars
+    ```scala
+    // in project/plugins.sbt:
+    addSbtPlugin("com.lightbend.akka.grpc" % "sbt-akka-grpc" % "akka.grpc.version$")
+    ```
+    @@@ 
+
+
+And then in your `build.sbt` file, you will need to enable the plugin for the project that contains the `.proto` files:
+
+sbt
+:   @@@vars
+    ```scala
+    // in build.sbt:
+    enablePlugins(AkkaGrpcPlugin)
+    ```
+    @@@
+
+And finally configure the language used in project:
+
+Scala
+:   @@@vars
+    ```scala
+    // in build.sbt:
+    akkaGrpcGeneratedLanguages := Seq(AkkaGrpc.Scala)
+    ```
+    @@@
+    
+Java
+:   @@@vars
+    ```scala
+    // in build.sbt:
+    akkaGrpcGeneratedLanguages := Seq(AkkaGrpc.Java)
+    ```
+    @@@
+
+After that, you should be able to generate and use the code to @ref[serve](serving-grpc.md) or @ref[consume](consuming-grpc.md) gRPC services.
```

Comment:
Same.

I would add a more _complete_ module creation:

```scala
lazy val `fancy-service-impl` = (project in file("fancy-impl"))
  .enablePlugins(LagomScala)
  .enablePlugins(AkkaGrpcPlugin)
  .enablePlugins(PlayAkkaHttp2Support) // enables serving HTTP/2
  .settings(
    akkaGrpcGeneratedLanguages := Seq(AkkaGrpc.Scala),
    akkaGrpcGeneratedSources :=
      Seq(
        AkkaGrpc.Server,
        AkkaGrpc.Client
      ),
    akkaGrpcExtraGenerators in Compile += PlayScalaServerCodeGenerator
   )
//...
```

Makes sense?

Reply:
Yes, I was in doubt about it also.